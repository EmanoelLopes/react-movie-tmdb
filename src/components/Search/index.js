import React from 'react';
import { StyledSearch } from './styles';

const Search = () => {
  return (
    <StyledSearch>
      <label>Pesquisar: </label>
      <input type="text" placeholder="Pesquisar tÃ­tulos..." />
    </StyledSearch>
  );
};

export default Search;
