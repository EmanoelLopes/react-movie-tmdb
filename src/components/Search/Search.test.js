import React from 'react';
import { mount } from 'enzyme';
import Search from 'components/Search';

let wrapped;
let defaultProps = {};

beforeEach(() => {
  wrapped = mount(
    <Search {...defaultProps} />
  );
});

afterEach(() => {
  wrapped.unmount();
});

describe('<Search />', () => {
  it('should match snapshot', () => {
    expect(wrapped).toMatchSnapshot();
  });
});
