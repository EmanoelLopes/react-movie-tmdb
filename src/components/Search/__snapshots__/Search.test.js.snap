// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Search /> should match snapshot 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
}

<Search>
  <styled.div>
    <StyledComponent
      forwardedComponent={
        Object {
          "$$typeof": Symbol(react.forward_ref),
          "attrs": Array [],
          "componentStyle": ComponentStyle {
            "componentId": "sc-bdVaJa",
            "isStatic": true,
            "lastClassName": "c0",
            "rules": Array [
              "
  display: flex;
",
            ],
          },
          "displayName": "styled.div",
          "foldedComponentIds": Array [],
          "render": [Function],
          "styledComponentId": "sc-bdVaJa",
          "target": "div",
          "toString": [Function],
          "warnTooManyClasses": [Function],
          "withComponent": [Function],
        }
      }
      forwardedRef={null}
    >
      <div
        className="c0"
      >
        <label>
          Pesquisar: 
        </label>
        <input
          placeholder="Pesquisar tÃ­tulos..."
          type="text"
        />
      </div>
    </StyledComponent>
  </styled.div>
</Search>
`;
